MvvmCross.Forms/Platform/Mac/Core/MvxFormsApplicationDelegate.cs

```diff
@@ -3,22 +3,61 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
+using System.Linq;
+using AppKit;
 using MvvmCross.Core;
+using MvvmCross.Exceptions;
 using MvvmCross.Forms.Presenters;
+using MvvmCross.IoC;
 using MvvmCross.Platform.Mac.Core;
+using MvvmCross.ViewModels;
 using Xamarin.Forms.Platform.MacOS;
 
 namespace MvvmCross.Forms.Platform.Mac.Core
 {
     public abstract class MvxFormsApplicationDelegate : FormsApplicationDelegate, IMvxApplicationDelegate
     {
+        private MvxFormsMacSetup _setup;
+        protected MvxFormsMacSetup Setup
+        {
+            get
+            {
+                if (_setup == null)
+                    _setup = CreateSetup(this, MainWindow);
+                return _setup;
```

Comment:
Can we make Mac work the same way as Android and iOS? (use a `InitializePlatform` method instead of ctor arguments) It seems cleaner to me to unify all setup changes on the same PR

Reply:
Done - I've made the changes but can someone please test both Mac and Mac Forms as I don't have access to a Mac this weekend.